scripts/generate_json_docs.py

```diff
@@ -290,10 +291,11 @@ def build_type(type_id, title, contents):
 
 def clean_source_path(source):
     source_path = ''
-    if 'gcloud-python' in source:
-        source_path = source.split('gcloud-python')[1]
+    for path in sys.path:
+        if path in source:
+            source_path = source.replace(path, '')
 
-    return source_path[1:]
+    return source_path
```

Comment:
This still looks wonky to me, but maybe I don't understance `clean_source_path`'s contract.

- What is supposed to happen if none of the `sys.path` entries is a prefix for `source`?
- What is supposed to happen if more than one entry is a prefix?